pkg/operation/update.go

```diff
@@ -28,13 +30,21 @@ func ParseUpdateOperation(request []byte, protocol protocol.Protocol) (*batch.Op
 		return nil, err
 	}
 
+	patches := operationData.Patches[0].GetStringValue(patch.PatchesKey)
+
```

Comment:
where do we lookup `ietf-json-patch` in the action field?

Reply:
Right now we support only one json patch - existing code. When I get to work on #155 (next week) I will add support for multiple patches in the operation and create document composer to vallidate and apply all patches.

Reply:
I have to do renaming before that since it impacts everything.